server/render/index.js

```diff
@@ -23,18 +21,18 @@ function bumpStat( group, name ) {
 	}
 }
 
-export function render( element ) {
-	memoizedRenderToString.cache = markupCache;
-
-	if ( ! memoizedRenderToString.cache.has( JSON.stringify( element ) ) ) {
-		bumpStat( 'calypso-ssr', 'loggedout-design-cache-miss' );
-	}
-
+export function render( element, path, initialState ) {
```

Comment:
Maybe better to have a `key` param instead of `path` and `initialState`, allowing caller to decide appropriate key?